docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,116 @@
+# .NET Standard vNext
+
+This document describes the plan for .NET Standard *vNext*, which includes the
+definition of its API surface.
+
+## Goals
+
+* **Include `Span<T>`**. In .NET Core 2.1 we've added `Span<T>` which is an
+  array like type that allows representing managed and unmanaged memory in a
+  uniform way and supports cheap slicing. It's at the heart of most
+  performance-related improvements in .NET Core 2.1 and allows managing buffers
+  in a more efficient way as it helps in reducing allocations and copying.  
+  `Span<T>` is considered a core type and requires runtime and compiler support
+  in order to be fully leveraged. Please note that when we refer to `Span<T>` in
+  this document, we really mean the family of span-related features which
+  includes other types such as `ReadOnlySpan<T>`, `Memory<T>`,
+  `ReadOnlyMemory<T>`, and the new `System.Buffers` namespace.
+
+* **Include core-APIs working with spans**. While `Span<T>` is available as a
+  .NET Standard compatible NuGet package (`System.Memory`) adding this package
+  cannot extend the members of .NET Standard types that deal with spans. For
+  example, .NET Core 2.1 added many APIs that allow working with spans, such as
+  `Stream.Read(Span<Byte>)`. Part of the value proposition to add span to .NET
+  Standard is to add the companion APIs as those cannot be added using a NuGet
+  package.
+
+* **General Goodness**. Since .NET Core was open sourced, we've added many small
+  features across the base class libraries such as `System.HashCode` for
+  combining hash codes or new overloads on `System.String`. There are about ~800
+  new members in .NET Core that are on types that are already in .NET Standard
+  2.0.
+
+## Decisions
+
+* We should **include** Brotli as `HttpClient` depends on it
+* We should **exclude** `DataAnnotations` (out-of-band today)
```

Comment:
Definitely better than OOB, but just to make sure I understand this: these are packages already provided on top of .NET Standard, and are serviced via independent releases, and we want to keep them that way, at least for now? That makes sense to me.

Reply:
That's correct.